### DISCUSSION TITLE
ADB Add simulated display

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#538] [Q&A] [answered]
hey, I know that you can create a virtual display with the ADB command `settings put global overlay_display_devices 1280x720/125`, but is there a way to add a display? because the command only works once.

**Comment by yume-chan** [comment]
Add another one? `settings put global overlay_display_devices "1280x720/125;1280x720/125"`

**Reply by CorruptionVR** [reply]
I mean, say you already used that command, is there a command that you can use, or an argument on that command that will add a display without destroying the already existing one

**Reply by yume-chan** [reply]
I'm not aware of any method to achieve that.

**Reply by CorruptionVR** [reply]
oh, so it would be impossible to create displays on demand to create per-app windows?

**Reply by yume-chan** [reply]
That's possible with https://github.com/Genymobile/scrcpy/issues/1887#issuecomment-1405184608. It's almost another use case so I don't know will Scrcpy integrate this. But I also don't want to maintain my own Java Android application. (I know no Android programming)

**Comment by CorruptionVR** [comment, marked as answer]
this has become an experimental feature in the new web app
